can't compile the library

When trying to compile the library, I face this error:

Now type `make' to compile xapp
KDB@KDB-laptop ~/Sandbox/xapps $ make
make  all-recursive
make[1]: Entering directory '/home/KDB/Sandbox/xapps'
Making all in libxapp
make[2]: Entering directory '/home/KDB/Sandbox/xapps/libxapp'
  CC       xapp-display.lo
xapp-display.c: In function 'create_blanking_window':
xapp-display.c:86:5: warning: 'gtk_widget_modify_bg' is deprecated (declared at /usr/include/gtk-3.0/gtk/deprecated/gtkstyle.h:753): Use 'gtk_widget_override_background_color' instead [-Wdeprecated-declarations]
     gtk_widget_modify_bg(window, GTK_STATE_NORMAL, &color);
     ^
xapp-display.c: In function 'xapp_display_blank_other_monitors':
xapp-display.c:104:86: warning: passing argument 1 of 'gtk_widget_get_window' from incompatible pointer type
     active_monitor = gdk_screen_get_monitor_at_window (screen, gtk_widget_get_window(window));
                                                                                      ^
In file included from /usr/include/gtk-3.0/gtk/gtkapplication.h:27:0,
                 from /usr/include/gtk-3.0/gtk/gtkwindow.h:33,
                 from /usr/include/gtk-3.0/gtk/gtkdialog.h:33,
                 from /usr/include/gtk-3.0/gtk/gtkaboutdialog.h:30,
                 from /usr/include/gtk-3.0/gtk/gtk.h:31,
                 from xapp-display.c:9:
/usr/include/gtk-3.0/gtk/gtkwidget.h:927:23: note: expected 'struct GtkWidget *' but argument is of type 'struct GtkWindow *'
 GdkWindow           * gtk_widget_get_window             (GtkWidget    *widget);
                       ^
xapp-display.c:108:5: error: 'for' loop initial declarations are only allowed in C99 or C11 mode
     for (int i = 0; i < self->priv->num_outputs; i++)
     ^
xapp-display.c:108:5: note: use option -std=c99, -std=gnu99, -std=c11 or -std=gnu11 to compile your code
xapp-display.c: In function 'xapp_display_unblank_monitors':
xapp-display.c:132:5: error: 'for' loop initial declarations are only allowed in C99 or C11 mode
     for (int i = 0; i < self->priv->num_outputs; i++)
     ^
Makefile:549: recipe for target 'xapp-display.lo' failed
make[2]: *** [xapp-display.lo] Error 1
make[2]: Leaving directory '/home/KDB/Sandbox/xapps/libxapp'
Makefile:464: recipe for target 'all-recursive' failed
make[1]: *** [all-recursive] Error 1
make[1]: Leaving directory '/home/KDB/Sandbox/xapps'
Makefile:395: recipe for target 'all' failed
make: *** [all] Error 2

--- a/libxapp/xapp-display.c
+++ b/libxapp/xapp-display.c
@@ -94,6 +94,7 @@ xapp_display_blank_other_monitors (XAppDisplay *self,
 {
     GdkScreen *screen;
     int active_monitor;
+    int i;
 
     g_return_if_fail (XAPP_IS_DISPLAY (self));
 
@@ -105,7 +106,7 @@ xapp_display_blank_other_monitors (XAppDisplay *self,
     self->priv->num_outputs = gdk_screen_get_n_monitors (screen);
     self->priv->windows = g_new (GtkWidget *, self->priv->num_outputs);
 
-    for (int i = 0; i < self->priv->num_outputs; i++)
+    for (i = 0; i < self->priv->num_outputs; i++)
     {
         if (i != active_monitor)
         {
@@ -124,12 +125,13 @@ xapp_display_blank_other_monitors (XAppDisplay *self,
 void
 xapp_display_unblank_monitors (XAppDisplay *self)
 {
+    int i;
     g_return_if_fail (XAPP_IS_DISPLAY (self));
 
     if (self->priv->windows == NULL)
         return;
 
-    for (int i = 0; i < self->priv->num_outputs; i++)
+    for (i = 0; i < self->priv->num_outputs; i++)
     {
         if (self->priv->windows[i] != NULL)
         {

